tldw_Server_API/tests/Services/test_shutdown_coordinator.py

```
@@ -0,0 +1,921 @@
+async def test_transport_registry_components_are_visible_to_shutdown_coordinator() -> None:
+    from tldw_Server_API.app.services.shutdown_transport_registry import (
+        ShutdownTransportRegistry,
+        build_shutdown_components,
+    )
+
+    events: list[str] = []
+    registry = ShutdownTransportRegistry()
+    registry.register_family(
+        "mcp.websocket",
+        active_count=lambda: 2,
+        drain=lambda timeout_s=None: events.append("mcp"),
```

Ruff (0.15.7)

[warning] Unused lambda argument: `timeout_s`

(ARG005)